.github/workflows/packages.yaml

```diff
@@ -45,8 +45,8 @@ jobs:
           '/src/ci/package.sh'
       env:
         ARCH: "${{ matrix.arch }}"
-        PACKAGE_VERSION: '1.2.1'
-        PACKAGE_RELEASE: '1'
+        PACKAGE_VERSION: '1.3.0'
+        PACKAGE_RELEASE: 'dev'
```

Comment:
https://github.com/Azure/iot-identity-service/commit/997ab230b78c6951bbe28b45f3ff08445666a1ee :)

(Maybe add a YAML comment so that we don't forget in the future.)

Reply:
Fixed

Reply:
Added comment.